dm/pkg/terror/error_list.go

```diff
@@ -624,6 +624,7 @@ const (
 	codeSchedulerWorkerExist
 	codeSchedulerWorkerNotExist
 	codeSchedulerWorkerOnline
+	codeSchedulerWorkerNotFree
```

Comment:
In order to keep the error code doesn't change between versions, we should append it to the end of this section.

Also, `make terror_check` and git add and commit the generated error code files